desktop/components/main_layout/templates/index.jade

```diff
@@ -5,8 +5,8 @@ block locals
 doctype html
 html(
   data-useragent=userAgent
-  data-user-type=sd.CURRENT_USER && sd.CURRENT_USER.type
-  data-lab-features=sd.CURRENT_USER && sd.CURRENT_USER.lab_features && sd.CURRENT_USER.lab_features.join(',')
+  data-user-type=sd && sd.CURRENT_USER && sd.CURRENT_USER.type
```

Comment:
So I received this error (the one we were seeing in prod where `sd` was not defined) locally... I'm not sure if this is the actual fix or just a bandaid.